#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raises a NameError with a custom message."""
    raise NameError(message)
